metadata-editor/app/model/UsageRightsProperty.scala

```diff
@@ -48,6 +48,11 @@ object UsageRightsProperty {
       UsageRightsProperty("supplier", "Supplier", "string", true, Some(UsageRightsConfig.freeSuppliers.sortWith(_.toLowerCase < _.toLowerCase))),
       UsageRightsProperty("suppliersCollection", "Collection", "string", false)
     )
+    
+    case _:CommissionedAgency => List(
+      UsageRightsProperty("supplier", "Supplier", "string", true),
+      UsageRightsProperty("suppliersCollection", "Collection", "string", false)
+    )
```

Comment:
We can do a good cleanup of those in the Picdar export (which is what this work is actually for).